research/CongressKeynote/Congress-Keynote.md

```diff
@@ -0,0 +1,466 @@
+
+You can see an interactive version of this slide [here](http://www.ecosounds.org/Zoom/1-top). 
+
+---
+
+## Slide 12: LDFC spectrograms of three sites at different latitudes
+{% include figure.html src="Slides/Slide12.png" caption="Slide 12." %}
+
+This slide compares three 24-hour, false-colour spectrograms of three soundscapes from different latitudes. All these recordings were obtained in the first week of July (winter) 2015. The top PNG recording is dominated by insects. It is hard to imagine stuffing more sound into this soundscape. PNG jungles are very loud. The middle recording (from Gympie National Park, about 100 kilometers north of Brisbane) is very quiet at night (because the temperatures are near freezing) and dominated by birds during the day. The bottom recording is from Sturt National Park in the far north-west corner of New South Wales, Australia. The conditions are desert-like. Not only is it very cold at night, but during the day it blows a gale (the yellow-red lines in the middle of the spectrogram). The desert morning chorus is diminutive. *Soundmarks* are very useful (actually essential) in order to orient oneself in a soundscape. They are the equivalent of *landmarks* in a landscape.
+
+---
+
+## Slide 13: Comparing land-use soundscapes
+{% include figure.html src="Slides/Slide14.png" caption="Slide 14." %}
+
+It is not very clever to distinguish a jungle soundscape from a desert soundscape -- it does not require LDFC spectrograms. This technique comes into its own when comparing soundscapes for diagnostic purposes, for example, when comparing the soundscapes resulting from different land use conditions.    
+
+This slide compares the soundscapes of several sites in the Adelbert Mountains of PNG, subject to different land uses, such as conservation, gardening and cocao plantation. The 24-hour LDFC spectrograms are aligned midnight to midnight. It is easily apparent that the garden site has a different soundscape.
+
+Note that an LDFC spectrogram is not an end in itself. Rather it facilitates the asking of further questions. An image does not easily lend itself to statistical analysis, but these LDFC spectrograms are derived from acoustic indices, which can be subject to statistical analysis. 
+
+---
+
+## Slide 14: Sampled recordings
+{% include figure.html src="Slides/Slide15.png" caption="Slide 15." %}
+
+Sampled recordings do not lend themselves to useful LDFC spectrograms. In this example the recording unit was set to record 10 minutes in every hour, with a longer recording interval during the morning and evening chorus. However note that final recording result actually missed most of the morning chorus!
```

Comment:
However, note that the final recording result missed most of the morning chorus!